yt_dlp/extractor/freetv.py

```diff
@@ -0,0 +1,176 @@
+import re
+
+from .common import InfoExtractor
+from ..utils import (
+    determine_ext,
+    ExtractorError,
+    int_or_none,
+    str_or_none,
+    traverse_obj,
+    urlencode_postdata,
+)
+
+
+class FreeTvBaseIE(InfoExtractor):
+    def _get_api_response(self, content_id, postdata, resource_type):
+        response = self._download_json(
+            'https://www.freetv.com/wordpress/wp-admin/admin-ajax.php',
+            content_id, data=urlencode_postdata(postdata), fatal=True,
+            note=f'Downloading {content_id} {resource_type} JSON')
+
+        if response.get('data') is False:
+            raise ExtractorError(f"Response doesn't contain {resource_type} data")
```

Comment:
`Response doesn't contain ...` means nothing to the user. So I don't understand why this error message is needed (as opposed to the default error of `reponse['data']`. But if u r doing it, it should be 
```suggestion
        if not response.get('data'):
            raise ExtractorError(f"Response doesn't contain {resource_type} data")
```